include/session/config/profile_pic.hpp

```diff
@@ -1,39 +1,57 @@
 #pragma once
 
+#include <stdexcept>
+
 #include "session/types.hpp"
 
 namespace session::config {
 
-// Profile pic info.  Note that `url` is null terminated (though the null lies just beyond the end
-// of the string view: that is, it views into a full std::string).
+// Profile pic info.
 struct profile_pic {
-  private:
-    std::string url_;
-    ustring key_;
+    static constexpr size_t MAX_URL_LENGTH = 223;
```

Comment:
given changes from c header are applied, then this can be:

`static constexpr size_t MAX_URL_LENGTH = _PROFILE_PIC_MAX_URL_SIZE_VAL;`